src/mapml/control/Control.Home.js

```diff
@@ -0,0 +1,63 @@
+export var HomeButton = L.Control.extend({
+  options: {
+    position: 'topleft',
+  },
+
+  onAdd: function (map) {
+    let container = L.DomUtil.create("div", "mapml-home-button leaflet-bar");
+
+    let link = L.DomUtil.create("a", "mapml-home-button", container);
+    link.innerHTML = "&#x2302";
+    link.href = "#";
+    link.title = "Home";
+    link.setAttribute('role', 'button');
+    link.setAttribute('aria-label', "Home");
```

Comment:
I understand that it's probably a convention to name a button with the to functionality reset to the initial location/starting point "home", but in the context of a web map I can see how this could be confusing to users with screen readers, they may interpret this as a button for location their own home. Can we use something else?

<!--

I would suggest using something other than "home", 

-->

Reply:
The intent of the 'home' button (a little 'house' icon is a common visual metaphor these days) is to reset the map to the original zoom / lat / lon **maybe** without changing other conditions (such as opacities, new / added layers etc., tbdiscussed).  What would be an appropriate role/label for such a function?

/cc @shepazu

Reply:
> What would be an appropriate role/label for such a function?

The best I can think of is "Reset to initial location/view". "location" doesn't make much sense in non-geo maps though.

Reply:
In the context menu its known as reload, would that work?